alpine/entrypoint.sh

```diff
@@ -90,6 +90,11 @@ fi
 
 if [[ $KUBERNETES ]]; then
     cp /opt/datadog-agent/agent/conf.d/kubernetes.yaml.example /opt/datadog-agent/agent/conf.d/kubernetes.yaml
+
+    # enable event collector
```

Comment:
could you add a warning here (in both entrypoints) that it needs to be enabled only for one node?
I'd rather have this warning in as many places as possible.